packages/common/src/config.ts

```diff
@@ -140,6 +154,12 @@ export function getConfig(): Config {
           .parse(process.env.REACT_APP_EXPLORER_DISABLE_MATCHING_ESTIMATES)
       ),
     },
+    allo: {
+      version: z
+        .string()
```

Comment:
would be better to use an zod enum here to protect against typos and other mistakes: https://github.com/colinhacks/zod#zod-enums

and define version as just "allo-v1" | "allo-v2"